### PR TITLE
fix: update pricing

### PR DESCRIPTION
**Pull request by casserni**
- fixes https://github.com/stoplightio/stoplight.io/issues/1102

### CODE REVIEW

File: netlify/landings/hubs.yaml [hunk 1]
```diff
@@ -80,7 +80,7 @@ pricingSection:
   cards:
     - icon: ['fad', 'user-check']
       title: "Standard"
-      tag: "$179 / month"
+      tag: "$149 / month"
       buttonText: "Free Trial. No CC Required."
       href: "https://next.stoplight.io"
       features: ['OpenAPI v2 & v3', 'HTTP Request Maker', 'Unlimited Published Pages']
```
Comment (lottamus) [inline netlify/landings/hubs.yaml]:
```suggestion
      features: ['Custom Domains', 'Basic Theming', 'Build History & Instant Rollbacks']
```

**Comment by philsturgeon** [comment]
@rossmcdonald you were asking about pricing, does this fix the pricing inconsistency you were talking about? If so, shall we merge it? I have a PR which needs to go up asap too so we can get a develop > master merge going. 

@marbemac

**Comment by rossmcdonald** [comment]
@philsturgeon No, this adds more confusion. We should meet to all get on the same page before this is merged.

**Comment by casserni** [comment]
These prices should be matching to the new pricing models for NEXT. Is the confusion that they don't align with prices for the new studio/platform?

**Comment by philsturgeon** [comment]
@rossmcdonald @casserni did this get discussed anywhere else?

**Comment by rossmcdonald** [comment]
@philsturgeon Nope. This still differs from what's displayed in the Stoplight app itself, so we should hold off on this (or close this PR) until we all get on the same page.

**Pull request closed without merging**